src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
+table.tHead.rows[0].cells[3].after(th);
+table.tFoot.rows[0].cells[3].after(th.cloneNode(true));
```

Comment:
When you clone a node using `cloneNode(true)`, it clones the node and all of its descendants. This means that any event listeners or properties set on the original node will not be copied to the clone. If you need to maintain such properties or listeners, you will need to reapply them to the cloned node.